src/utilities/connectors.ts

```diff
@@ -25,12 +25,12 @@ const ledger = new LedgerConnector({
   pollingInterval: POLLING_INTERVAL,
 });
 
-export const ConnectorNames = {
-  Injected: 'MetaMask',
-  WalletConnect: 'WalletConnect',
-  BSC: 'BSC',
-  Ledger: 'Ledger',
-};
+export enum ConnectorNames {
+  Injected = 'MetaMask',
```

Comment:
I guess to be consistent with the rest of the enum we should update this to `MetaMask = 'MetaMask'`

Reply:
Feels a bit like expanding the scope of the PR but I have no problem with that change.

Reply:
Ok fair enough, in this case I'm cool with leaving this as it is and making the update later.

Reply:
I thought I pushed it actually